app/lib/auth/jwt.ts

```
@@ -0,0 +1,51 @@
+      (err, token) => {
+        if (err) reject(err);
+        else if (token) resolve(token);
+        else reject(new Error('Failed to generate token'));
+      },
+    );
+  });
+}
+
+export async function verifyJWT(token: string): Promise<JWTPayload> {
+  return new Promise((resolve, reject) => {
+    jwt.verify(token, JWT_SECRET!, (err, decoded) => {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)